runtime/src/transaction/runtime_transaction.rs

```diff
@@ -39,6 +39,7 @@ impl StaticMeta for RuntimeTransactionStatic {
 }
 
 impl RuntimeTransactionStatic {
+    #[allow(dead_code)]
```

Comment:
i think we can just `#![allow(dead_code)]` at the top of both files now? (or maybe just once in `transaction/mod.rs` :thinking:)

Reply:
I think you should be able to add it once in the runtime lib.rs file as an annotation on the module i.e.

```rust
#![allow(dead_code)]
mod runtime_transaction;
```

Reply:
Yea, my thought was these functions will most likely be put in use one at time, I'd just need to removed `#allow...` when that happen. Not having strong opinion here tho